shared/wallets/transaction-details/index.js

```diff
@@ -164,23 +168,31 @@ class TransactionDetails extends React.Component<Props> {
   }
 
   render() {
-    const {sender, receiver} = propsToParties(this.props)
```

Comment:
likely simpler to just make a wrapper that does this and renders 2 things. the loader and the previous version of this